src/targets/awsLambdaLayer.ts

```diff
@@ -1,25 +1,47 @@
 import * as fs from 'fs';
+import * as path from 'path';
+
+import * as Github from '@octokit/rest';
+import * as simpleGit from 'simple-git/promise';
+import { getGlobalGithubConfig } from '../config';
+import {
+  getAuthUsername,
+  getGithubApiToken,
+  getGithubClient,
+  GithubRemote,
+} from '../utils/githubApi';
+
 import { logger as loggerRaw } from '../logger';
-import { TargetConfig } from '../schemas/project_config';
+import { GithubGlobalConfig, TargetConfig } from '../schemas/project_config';
 import { BaseTarget } from './base';
 import { BaseArtifactProvider } from '../artifact_providers/base';
 import { ConfigurationError, reportError } from '../utils/errors';
 import {
-  AddLayerVersionPermissionRequest,
-  Lambda,
-  PublishLayerVersionCommandOutput,
-  PublishLayerVersionRequest,
-} from '@aws-sdk/client-lambda';
-import { DescribeRegionsCommandOutput, EC2 } from '@aws-sdk/client-ec2';
+  AwsLambdaLayerManager,
+  CompatibleRuntime,
+  extractRegionNames,
+  getAccountFromArn,
+  getRegionsFromAws,
+} from '../utils/awsLambdaLayerManager';
+import { createSymlinks } from '../utils/symlink';
+import { withTempDir } from '../utils/files';
+import { isDryRun } from '../utils/helpers';
 
 const logger = loggerRaw.withScope(`[aws-lambda-layer]`);
 
+const DEFAULT_REGISTRY_REMOTE: GithubRemote = new GithubRemote(
```

Comment:
This now duplicates the value in `src/targets/registry`, wdyt about extracting it as a constant/function somewhere?

Reply:
Extracted to `getRegistryGithubRemote` in `src/utils/githubApi`.